app/src/main/java/com/eventyay/organizer/core/event/list/EventListFragment.java

```diff
@@ -108,6 +109,10 @@ public CharSequence getPageTitle(int position) {
     @Override
     public void onStart() {
         super.onStart();
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            defaultViewOutlineProvider = getActivity().findViewById(R.id.appBar).getOutlineProvider();
+            getActivity().findViewById(R.id.appBar).setOutlineProvider(null);
+        }
```

Comment:
This is a workaround, not a fix

Reply:
I apologize for such a bad way of filing the PR. It's just because I'm not yet comfortable with git on my computer.

Reply:
I still can't understand that why can't this be a good fix. If the previous PR on this issue by @sridharjajoo {#1179} was having a problem, what could be the other aproach?

Reply:
Because this is not a bug in our code, hence should be fixed by chanes in XML layouts or styles, not by hacking the framework